m0t1fs/linux_kernel/file.c

```diff
@@ -3091,13 +3091,17 @@ static int pargrp_iomap_dgmode_recover(struct pargrp_iomap *map)
 		rc = -EIO;
 		goto end;
 	}
+
+#if 0 /* NA_FOR_INTEL_ISA */
```

Comment:
Why do you want to keep #if 0 code ?

Reply:
The code is currently kept just for future reference if required.  Ref https://github.com/Seagate/cortx-motr/pull/862#discussion_r676521454. But if everyone agrees that we should remove it, then I'll remove it.

Reply:
It is better to make the code clean by removing the #if 0.
If needed this code can be added later as it will be present in the older commits.

Reply:
Done.